doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -0,0 +1,599 @@
+   "ACPI S3 Support", "Intel Advanced Menu -> ACPI Settings", "Disabled"
+   "Native ASPM", "Intel Advanced Menu -> ACPI Settings", "Disabled"
+
+.. note:: BIOS settings depend on the platform and BIOS version; some may
+   not be applicable.
+
+Recommended kernel cmdline for RTVM
+-----------------------------------
+
+.. code-block:: none
+
+   root=root=PARTUUID=<UUID of rootfs partition> rw rootwait nohpet console=hvc0 console=ttyS0 \
+   no_timer_check ignore_loglevel log_buf_len=16M consoleblank=0 \
+   clocksource=tsc tsc=reliable x2apic_phys processor.max_cstate=0 \
+   intel_idle.max_cstate=0 intel_pstate=disable mce=ignore_ce audit=0 \
+   isolcpus=nohz,domain,1 nohz_full=1 rcu_nocbs=1 nosoftlockup idle=poll \
+   irqaffinity=0
+
+
+Configure RDT
+-------------
+
+In addition to setting the CAT configuration via HV commands, we allow
+developers to add CAT configurations to the VM config and configure
+automatically at the time of RTVM creation. Refer to :ref:`rdt_configuration`
+for details on RDT configuration and :ref:`hv_rdt` for details on RDT
+high-level design.
+
+Set up the core allocation for the RTVM
+---------------------------------------
+
+In our recommended configuration, two cores are allocated to the RTVM:
+core 0 for housekeeping and core 1 for RT tasks. In order to achieve
+this, follow the below steps to allocate all housekeeping tasks to core 0:
+
+#. Launch the RTVM::
+
+   # /usr/share/acrn/samples/nuc/launch_hard_rt_vm.sh
+
+#. Log in to the RTVM as root and run the script as below:
+
+   .. code-block:: bash
+
+      #!/bin/bash
+      # Copyright (C) 2019 Intel Corporation.
+      # SPDX-License-Identifier: BSD-3-Clause
+      # Move all IRQs to core 0.
+      for i in `cat /proc/interrupts | grep '^ *[0-9]*[0-9]:' | awk {'print $1'} | sed 's/:$//' `;
+      do
+          echo setting $i to affine for core zero
+          echo 1 > /proc/irq/$i/smp_affinity
+      done
+
+      # Move all rcu tasks to core 0.
+      for i in `pgrep rcu`; do taskset -pc 0 $i; done
+
+      # Change realtime attribute of all rcu tasks to SCHED_OTHER and priority 0
+      for i in `pgrep rcu`; do chrt -v -o -p 0 $i; done
+
+      # Change realtime attribute of all tasks on core 1 to SCHED_OTHER and priority 0
+      for i in `pgrep /1`; do chrt -v -o -p 0 $i; done
+
+      # Change realtime attribute of all tasks to SCHED_OTHER and priority 0
+      for i in `ps -A -o pid`; do chrt -v -o -p 0 $i; done
+
+      echo disabling timer migration
+      echo 0 > /proc/sys/kernel/timer_migration
+
+   .. note:: You can ignore the error messages during the script running.
+
+Run cyclictest
+--------------
+
+#. Refer to the :ref:`troubleshooting section <enabling the network on RTVM>`   below that discusses how to enable the network connection for RTVM.
+
+#. Launch the RTVM and log in as root.
+
+#. Install the ``cyclictest`` tool::
+
+   # swupd bundle-add dev-utils --skip-diskspace-check
```

Comment:
That'd be for a Clear Linux User VM but we use Ubuntu

Reply:
@fuzhongl Please respond.

Reply:
Please change 
#. Install the ``cyclictest`` tool::

   # swupd bundle-add dev-utils --skip-diskspace-check
to
#. Install the ``rt-tests`` tool::

   # sudo apt install rt-tests

Reply:
complete